### PR TITLE
PRIME-2138 Update provisioner link webpage

### PR DESCRIPTION
**Pull request by neophyte57**
FYI - some logic for the API changes (PRIME-2239) has been refactored to work with PRIME-2138. If possible, please retest PRIME-2239 as well. 
Overall changes:
- Rename PharmeNet ID to Global PharmaNet ID
- Add new fields: Access Type, College ID, Licence Number, PharmaNet ID
- Licence Number should be labeled differently based on the college code, Like CPSID Number, Registration Number etc. which should match the label in the enrolment screen. 
- If PharmaNet ID is not present, hide the field 
image
image

### CODE REVIEW

File: prime-dotnet-webapi/Models/EnrolmentCertificate.cs [hunk 1]
```diff
@@ -34,8 +37,33 @@ public static EnrolmentCertificate Create(Enrollee enrollee)
...
+                        LicenseCode = cert.LicenseCode,
+                        CollegeLicenseNumber = cert.LicenseNumber,
+                        PharmaNetId = cert.PractitionerId
+                    })
             };
         }
     }
+
+
+    public class EnrolleeCertDto
```
Comment (neophyte57) [inline prime-dotnet-webapi/Models/EnrolmentCertificate.cs:55]: Just a test ... will this comment remain after PR closed and re-opened?

File: prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html [hunk 2]
```diff
@@ -5,11 +5,11 @@
   <ng-container *ngIf="certificate; else noCertificate">
 
     <app-page-subheader>
-      <ng-container appPageSubheaderTitle>PharmaNet ID</ng-container>
+      <ng-container appPageSubheaderTitle>Global PharmaNet ID</ng-container>
     </app-page-subheader>
 
     <p class="mb-3">
-      The user's PharmaNet ID number (GPID) is:
+      The user's Global PharmaNet ID number (GPID) is:
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html:12]: Should "number" be removed?

File: prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html [hunk 3]
```diff
@@ -21,19 +21,38 @@
...
+    <ng-container *ngFor="let licence of certificate.licences">
+      <app-enrollee-property title="Licence"
+                             [makeBold]="true">
+        <app-enrollee-property title="College ID"
+                               [makeBold]="true">
+          {{ licence.practRefId }}
+        </app-enrollee-property>
+        <app-enrollee-property [title]="licence.collegeCode === CollegeLicenceClassEnum.CPSBC ? 'CPSID Number' :
+                                (licence.collegeCode === CollegeLicenceClassEnum.CDSBC
+                                || licence.collegeCode === CollegeLicenceClassEnum.OptometryBC) ? 'Registration Number' :
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html:43]: Missing `CollegeLicenceClassEnum.CPBC`

File: prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html [hunk 4]
```diff
@@ -21,19 +21,38 @@
...
-
     </div>
 
+    <ng-container *ngFor="let licence of certificate.licences">
+      <app-enrollee-property title="Licence"
+                             [makeBold]="true">
+        <app-enrollee-property title="College ID"
+                               [makeBold]="true">
+          {{ licence.practRefId }}
+        </app-enrollee-property>
```
Comment (neophyte57) [inline prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html:40]: For colleges that don't have a Pract Ref ID, can a "-" be displayed instead ... currently

image

Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html]: Good idea. Logic is added to show "-" when no college ID.

File: prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html (hunk 2 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html]: Reviewed the UI with Nelson but did not catch that. "Number" is removed.

File: prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html (hunk 3 above)
Comment (bergomi02) [reply prime-angular-frontend/src/app/modules/provisioner-access/pages/certificate/certificate.component.html]: Good catch. The missing code is added.

**Review by bergomi02 — APPROVED** [review]
All good!

**Pull request merged**